app/api/items/update_item.feature

```diff
@@ -0,0 +1,186 @@
+Feature: Update item
+
+Background:
+  Given the database has the following table 'users':
+    | ID | sLogin | tempUser | idGroupSelf | idGroupOwned |
+    | 1  | jdoe   | 0        | 11          | 12           |
+  And the database has the following table 'groups':
+    | ID | sName      | sType     |
+    | 11 | jdoe       | UserSelf  |
+    | 12 | jdoe-admin | UserAdmin |
+  And the database has the following table 'items':
```

Comment:
For a service updating the attributes and being able to show that some have changed, some haven't, I thing it would make sense to put value in the column... at least in some of them.

Reply:
fixed